server/helpers/utils.js

```diff
@@ -1,17 +1,47 @@
 import slug from 'slug';
 import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import env from 'dotenv';
 
+env.config();
 /**
  * @param {string} title
  * @return {string} unique_slug
  */
-const createUniqueSlug = title => `${slug(title, { lower: true })}-${Date.now()}`;
 
 /**
  * @param {string} password
  * @return {string} hash
  */
+const { SECRET, TOKENEXPIRYDATE, COOKIEEXPIRYDATE } = process.env;
```

Comment:
use underscore to space your constant names out `TOKEN_EXPIRY_DATE` and `COOKIE_EXPIRYDATE`